### PR TITLE
fix: ColorsUtil CSP issue

### PR DESCRIPTION
**Pull request by shuowu-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: assets/sass/okta-theme.scss [hunk 1]
```diff
@@ -756,6 +756,7 @@
     border-color: $disabled-primary-button-bg-color;
     background-image: none;
     box-shadow: none;
+    opacity: 0.5;
```
Comment (aarongranick-okta) [inline assets/sass/okta-theme.scss:759]: seems like there may be a small difference here. The "opacity" rule added by the "template" function is only applied if `colors.brand` is set, but the CSS rule here would apply in all cases.

File: src/util/ColorsUtil.js [hunk 2]
```diff
@@ -41,20 +28,16 @@ fn.lighten = function(hex, lum) {
...
-  main.appendChild(style);
-};
-
-fn.isLoaded = function() {
-  return !!document.getElementById(Enums.WIDGET_CONFIG_COLORS_ID);
+fn.addStyle = function($containerEl, colors) {
+  const $primaryButton = $containerEl.find('.button-primary');
+  $primaryButton
+    .css('background', colors.brand)
+    .on('mouseenter', function() {
```
Comment (aarongranick-okta) [inline src/util/ColorsUtil.js:35]: do we actually want "mouseover"?

File: src/util/ColorsUtil.js [hunk 3]
```diff
@@ -41,20 +28,16 @@ fn.lighten = function(hex, lum) {
...
+      $(this).css('background', fn.lighten(colors.brand, 0.05));
+    })
+    .on('mouseleave', function() {
```
Comment (aarongranick-okta) [inline src/util/ColorsUtil.js:38]: same question, do we want "mouseout" ?

File: src/v2/controllers/FormController.ts [hunk 4]
```diff
@@ -67,6 +68,14 @@ export default Controller.extend({
           currentViewState,
         }
       }).last();
+      
+      // Apply the custom colors after render
+      if (this.settings.get('colors.brand')) {
```
Comment (jaredperreault-okta) [inline src/v2/controllers/FormController.ts:73]: Do we no longer need the `!ColorUtils.isLoaded()` check?

File: src/v1/BaseLoginRouter.js [hunk 5]
```diff
@@ -216,6 +207,15 @@ export default Router.extend({
 
         this.controller.render();
 
+        // Apply the custom colors after render
+        if (this.settings.get('colors.brand')) {
```
[the remark this answers is not in the document]
Comment (shuowu-okta) [reply src/v1/BaseLoginRouter.js:211]: with this approach, not style block will be injected, then no need to check that any more

**Pull request closed without merging**